Source/Core/Common/HttpRequest.h

```diff
@@ -35,6 +35,13 @@ class HttpRequest final
   using Response = std::optional<std::vector<u8>>;
   using Headers = std::map<std::string, std::optional<std::string>>;
 
+  struct Multiform
+  {
+    std::string name;
+    std::string data;
+    size_t size;
```

Comment:
Does `data.size() != size` ever happen?

Reply:
Shouldn't happen, that being said I should probably just use `data.size()` instead of having the size field.